src/screens/Departures/DeparturesOnboardingScreen.tsx

```diff
@@ -0,0 +1,93 @@
+import {useTranslation} from '@atb/translations';
+import ThemeText from '@atb/components/text';
+import Button from '@atb/components/button';
+import {ScrollView, View} from 'react-native';
+import React from 'react';
+import {StyleSheet} from '@atb/theme';
+import {Confirm} from '@atb/assets/svg/mono-icons/actions';
+import {StaticColorByType} from '@atb/theme/colors';
+import DeparturesTexts from '@atb/translations/screens/Departures';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {DeparturesStackProps} from '@atb/screens/Departures/types';
+import storage from '@atb/storage';
+import useFocusOnLoad from '@atb/utils/use-focus-on-load';
+import {screenReaderPause} from '@atb/components/accessible-text';
+import {DeparturesOnboarding} from '@atb/assets/svg/color/images';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+type Props = DeparturesStackProps<'DeparturesOnboardingScreen'>;
+
+export const DeparturesOnboardingScreen = ({navigation}: Props) => {
+  const {t} = useTranslation();
+  const styles = useStyles();
+  const focusRef = useFocusOnLoad();
+
+  return (
+    <SafeAreaView edges={['top', 'left', 'right']} style={styles.container}>
+      <ScrollView
+        contentContainerStyle={styles.scrollViewContent}
+        bounces={false}
+      >
+        <View
+          accessible={true}
+          accessibilityLabel={t(DeparturesTexts.onboarding.a11yLabel)}
+          ref={focusRef}
+        >
+          <ThemeText
+            style={styles.title}
+            color={themeColor}
+            type="heading--big"
+            accessible={true}
+          >
+            {t(DeparturesTexts.onboarding.title)}
+          </ThemeText>
+          <ThemeText color={themeColor} style={styles.body}>
+            {t(DeparturesTexts.onboarding.body.part1)}
+          </ThemeText>
+          <View style={styles.illustration}>
+            <DeparturesOnboarding />
+          </View>
+          <ThemeText color={themeColor} style={styles.body} isMarkdown={true}>
+            {t(DeparturesTexts.onboarding.body.part2)}
+          </ThemeText>
+        </View>
+        <Button
+          text={t(DeparturesTexts.onboarding.button)}
+          onPress={async () => {
+            await storage.set(
+              '@ATB_has_read_departures_v2_onboarding',
```

Comment:
It would be nice if we have this value inside something else and not here directly in the view, I saw other places uses a concept like StoreKey. I think is better to change this value in one place than searching in all the source code in the future.

Reply:
Good suggestion! 👍 I created a new enum key for this in the storage-file now. I will for now not "move" the other storage keys into the new enum, as it would require more regression testing.